### PR TITLE
added carousel files

### PR DESCRIPTION
**Pull request by ElenaLadutska**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <img src="./assets/1.png" alt="first image of Stitch" id="first_img" class="fade">
+        <img src="./assets/2.png" alt="second image of Stitch" id="second_img" class="fade">
+        <img src="./assets/3.png" alt="third image of Stitch" id="third_img" class="fade">
+        <img src="./assets/4.png" alt="fourth image of Stitch" id="fourth_img" class="fade">
+        <img src="./assets/5.png" alt="fifth image of Stitch" id="fifth_img" class="fade">
+        <img src="./assets/6.png" alt="sixth image of Stitch" id="sixth_img" class="fade">
+      </div>
+      <div class="rightArrow">
+        <p> > </p>
+      </div>
```
Comment (kathrynowy) [inline index.html]: create imgs with js
add a variable which will tell how many img you will have

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      <div class="images">
+        <img src="./assets/1.png" alt="first image of Stitch" id="first_img" class="fade">
+        <img src="./assets/2.png" alt="second image of Stitch" id="second_img" class="fade">
+        <img src="./assets/3.png" alt="third image of Stitch" id="third_img" class="fade">
+        <img src="./assets/4.png" alt="fourth image of Stitch" id="fourth_img" class="fade">
+        <img src="./assets/5.png" alt="fifth image of Stitch" id="fifth_img" class="fade">
+        <img src="./assets/6.png" alt="sixth image of Stitch" id="sixth_img" class="fade">
+      </div>
+      <div class="rightArrow">
+        <p> > </p>
```
Comment (kathrynowy) [inline index.html]: do ypu need this p?

File: index.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+const images = document.getElementsByClassName('images')[0];
+const leftArrow = document.getElementsByClassName('leftArrow')[0];
+const rightArrow = document.getElementsByClassName('rightArrow')[0];
+const pages = document.getElementsByClassName('pages')[0];
+
+const imagesArray = [];
+const pagesArray = [];
+
+let startedIndex = 0;
```
Comment (kathrynowy) [inline index.js]: startED?

File: index.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+for (let i = 0; i < images.children.length; i++) {
+  imagesArray.push(images.children[i]);
+}
+
+for (let i = 0; i < pages.children.length; i++) {
+  pagesArray.push(pages.children[i]);
+}
```
Comment (kathrynowy) [inline index.js]: won't they be the same?

File: index.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+pagesArray[startedIndex].classList.add('active');
+
+leftArrow.addEventListener('click', function(){
+  if (startedIndex <= 0) {
+    startedIndex = startedIndex + 1;
```
Comment (kathrynowy) [inline index.js]: startedIndex++;

File: index.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  };
+  
+  imagesArray[startedIndex].style.display = 'none';
+  pagesArray[startedIndex].classList.remove('active');
+
+  startedIndex = startedIndex - 1;
```
Comment (kathrynowy) [inline index.js]: .

File: index.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  };
+
+  imagesArray[startedIndex].style.display = 'none';
+  pagesArray[startedIndex].classList.remove('active');
+
+  startedIndex = startedIndex + 1;
+
+  pagesArray[startedIndex].classList.add('active');
+  imagesArray[startedIndex].style.display = 'flex';
+});
```
Comment (kathrynowy) [inline index.js]: can you reuse this function? 
just pass a param or something?

make it not via eventListener, by adding it to onclick on html

Comment (kathrynowy) [reply index.js]: function to onClick

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (ElenaLadutska) [reply index.html]: yes, because I need to show > here and in the left arrow too I need <

File: index.js (hunk 4 above)
Comment (ElenaLadutska) [reply index.js]: yeah, they will

File: index.js (hunk 5 above)
Comment (ElenaLadutska) [reply index.js]: fixed

File: index.js (hunk 3 above)
Comment (ElenaLadutska) [reply index.js]: renamed

File: index.js (hunk 6 above)
Comment (ElenaLadutska) [reply index.js]: fixed

File: index.js (hunk 7 above)
Comment (ElenaLadutska) [reply index.js:47]: I created 2 functions.

Comment (ElenaLadutska) [reply index.js:47]: fixed

Comment (ElenaLadutska) [reply index.js]: fixed

File: index.html (hunk 1 above)
Comment (ElenaLadutska) [reply index.html]: fixed

**Pull request merged**